.circleci/config.yml

```
@@ -16,20 +16,21 @@ jobs:
             echo 'export PATH=~/.npm-global/bin:$PATH' >> $BASH_ENV
             echo 'export NODE_PATH=~/.npm-global/lib/node_modules:$NODE_PATH' >> $BASH_ENV
 
+      - run:
+          name: "Check git conflicts"
+          command: |
+            git remote add upstream https://github.com/pingcap/docs-cn.git
+            git fetch upstream
+            python3 scripts/check-conflicts.py $(git diff-tree --name-only --no-commit-id -r upstream/master..HEAD -- '*.md' ':(exclude).github/*')
+      
       - run:
           name: "Install markdownlint"
           command: |
             npm install -g markdownlint-cli@0.17.0
 
-      - run:
-          name: "Lint README"
-          command: markdownlint README.md
-
       - run:
           name: "Lint edited files"
           command: |
-            git remote add upstream https://github.com/pingcap/docs-cn.git
-            git fetch upstream
             markdownlint $(git diff-tree --name-only --no-commit-id -r upstream/master..HEAD -- '*.md' ':(exclude).github/*' ':(exclude)v1.0/*' ':(exclude)v2.0/*' ':(exclude)v2.1-legacy/*')
```

```suggestion
            markdownlint $(git diff-tree --name-only --no-commit-id -r upstream/master..HEAD -- '*.md' ':(exclude).github/*')
```
ditto for line 49

You can just copy/paste the yaml file from the `docs` repo and revert some unnecessary changes from there.